tools/data-handler/src/interfaces/project-interfaces.ts

```diff
@@ -115,97 +130,98 @@ export interface linkType {
 }
 
 // Project metadata details.
-export interface project {
+export interface ProjectMetadata {
   name: string;
   path: string;
   prefix: string;
   numberOfCards: number;
 }
 
 // Project resource, such as workflow, template or card type
-export interface resource {
+export interface Resource {
   name: string;
   path?: string;
 }
 
 // Template content.
-export interface templateContent {
-  name: string;
-  cards?: card[];
-}
+// todo: not used???
```

Comment:
If there is some reason why this is apparently unused, I can keep it. Otherwise, before merging this PR, I will delete it.